lib/ansible/plugins/lookup/ini.py

```diff
@@ -15,7 +15,10 @@
       - "You can also read a property file which - in this case - does not contain section."
     options:
       _terms:
-        description: The key(s) to look up
+        description:
+            The key(s) to look up. On Python 2, key names are case B(insensitive).
+            In Python 3, key names are case B(sensitive). Duplicate key names found
```

Comment:
this wont run on py2 anymore .. specially with the string syntax below

you might want to make separate PR to port the py2 info back